bin/node/runtime/Cargo.toml

```diff
@@ -36,10 +37,11 @@ sp-keyring = { version = "3.0.0", optional = true, path = "../../../primitives/k
 sp-session = { version = "3.0.0", default-features = false, path = "../../../primitives/session" }
 sp-transaction-pool = { version = "3.0.0", default-features = false, path = "../../../primitives/transaction-pool" }
 sp-version = { version = "3.0.0", default-features = false, path = "../../../primitives/version" }
+sp-npos-elections = { version = "3.0.0", default-features = false, path = "../../../primitives/npos-elections" }
```

Comment:
What does this pallet responsible for?

Reply:
It used for making the number of nominations configurable

Reply:
In what version Polkadot did add it?

Reply:
0.8.30